test/connectivity/scripts/connectivityTest.sh

```diff
@@ -113,7 +120,9 @@ function process_args() {
                 '-iotHubConnectionString' ) saveNextArg=7;;
                 '-eventHubConnectionString' ) saveNextArg=8;;
                 '-eventHubConsumerGroupId' ) saveNextArg=9;;
-                '-loadGenMessageFrequency' ) saveNextArg=10;;
+                '-testDuration' ) saveNextArg=10;;
+                '-loadGenMessageFrequency' ) saveNextArg=11;;
+
```

Comment:
Is this whitespace intentional?

Reply:
I followed what we did in runE2ETests.sh; and I used ShellCheck online to verify, and it doesn't complain this space.  I will keep it for now.